readthedocs/templates/core/build-overview.md

```diff
@@ -0,0 +1,34 @@
+{% comment %}
+Template used for generating the build overview page that is posted as a comment on the build PR.
+
+Whitespaces and newlines are important in some places like lists and tables,
+make sure to adjust the tags accordingly, as they introduce newlines.
+{% endcomment %}
+## Documentation build overview
+
+> 📚 [{{ project.name }}](https://{{ PRODUCTION_DOMAIN }}{% url "projects_detail" project.slug %}) | 🛠️ build [#{{ current_version_build.pk }}](https://{{ PRODUCTION_DOMAIN }}{% url "builds_detail" project.slug current_version_build.pk %}) ({{ current_version_build.commit }}) | 🔍 [preview]({{ current_version.get_absolute_url }})
```

Comment:
I'd probably remove the commit here if we aren't planning to update this comment with new content on each build yet.

Reply:
Yes. I also mentioned that it would be easier to read/parse if we use bullets here with one link per bullet: https://github.com/readthedocs/readthedocs.org/pull/12230#discussion_r2161346501

Reply:
The comment is updated on each build, so it's useful to see from which commit the overview is.